src/utility.ts

```diff
@@ -239,24 +237,16 @@ export const Utility = {
 		return toReturn;
 	},
 
-	_toJSONSchema<T>(
-		collectionName: string,
-		collectionType: CollectionType,
-		schema: TigrisSchema<T> | TigrisTopicSchema<T>
-	): string {
+	_toJSONSchema<T>(collectionName: string, schema: TigrisSchema<T>): string {
 		const root = {};
 		const pkeyMap = {};
 		const keyMap = {};
 		root["title"] = collectionName;
 		root["additionalProperties"] = false;
 		root["type"] = "object";
 		root["properties"] = this._getSchemaProperties(schema, pkeyMap, keyMap);
-		root["collection_type"] = collectionType;
-		if (collectionType === "documents") {
-			Utility._postProcessDocumentSchema(root, pkeyMap);
-		} else if (collectionType === "messages") {
-			Utility._postProcessMessageSchema(root, keyMap);
-		}
+		root["collection_type"] = "documents";
```

Comment:
should not be needed.

Reply:
remind me - server defaults to documents?

Reply:
that's right